gaphor/SysML/blocks/property.py

```diff
@@ -0,0 +1,51 @@
+"""Property item."""
+
+from gaphor import UML
+from gaphor.core.modeling.properties import attribute
+from gaphor.diagram.presentation import ElementPresentation, Named
+from gaphor.diagram.shapes import Box, EditableText, Text, draw_border
+from gaphor.diagram.support import represents
+from gaphor.diagram.text import FontWeight, VerticalAlign
+from gaphor.UML.classes.stereotype import stereotype_compartments
+
+
+@represents(UML.Property)
+class PropertyItem(ElementPresentation, Named):
```

Comment:
`EleenmentPresentation` can be typed (`ElementPresentation[UML.Property]`). Then the subject field will have the right type.